streaming/pom.xml

```diff
@@ -66,6 +66,10 @@
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-servlet</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.reactivestreams</groupId>
+      <artifactId>reactive-streams</artifactId>
+    </dependency>
```

Comment:
Please do no add additional dependencies to streaming module. Any additional dependencies can be separate modules